pymongo/errors.py

```diff
@@ -167,6 +170,15 @@ def details(self):
         """
         return self.__details
 
+    @property
+    def max_wire_version(self):
```

Comment:
Can we make this private+undocumented (`_ max_wire_version`)? Do you see any reason to make it public?

Reply:
Can you make this private (again)? Somehow it got re-added.